README.md

```diff
@@ -1,6 +1,35 @@
 ![alt text](https://fourtf.com/img/chatterino-icon-64.png)
-Chatterino 2 [![GitHub Actions Build (Windows, Ubuntu, MacOS)](https://github.com/Chatterino/chatterino2/workflows/Build/badge.svg?branch=master)](https://github.com/Chatterino/chatterino2/actions?query=workflow%3ABuild+branch%3Amaster) [![Cirrus CI Build (FreeBSD only)](https://api.cirrus-ci.com/github/Chatterino/chatterino2.svg?branch=master)](https://cirrus-ci.com/github/Chatterino/chatterino2/master) [![Chocolatey Package](https://img.shields.io/chocolatey/v/chatterino?include_prereleases)](https://chocolatey.org/packages/chatterino) [![Flatpak Package](https://img.shields.io/flathub/v/com.chatterino.chatterino)](https://flathub.org/apps/details/com.chatterino.chatterino)
+Chatterino 2 [![GitHub Actions Build (Windows, Ubuntu, MacOS)](https://github.com/SevenTV/chatterino7/workflows/Build/badge.svg?branch=master)](https://github.com/SevenTV/chatterino7/actions?query=workflow%3ABuild+branch%3Amaster) [![Chocolatey Package](https://img.shields.io/chocolatey/v/chatterino7?include_prereleases)](https://chocolatey.org/packages/chatterino7)
 ============
+## Chatterino7
+
+Chatterino7 is a fork of Chatterino 2 that says in line with Chatterino 2 but adds features from 7TV, that aren't supported by regular Chatterino.
+
+### Features of Chatterino7
```

Comment:
Maybe this could be a table like this:

|Feature| Chatterino 7 | Chatterino 2 |
|---|:---:|:---:|
| 7TV Global Emotes | ✅ |✅ |
| 7TV Channel Emotes | ✅ |✅ |
| 7TV Emote Updates | ✅ |✅ |
| 7TV Badges | ✅ | ✅ * |
| 7TV Personal Emotes | ✅ | ❌  |
| 7TV Name Paints | ✅ | ❌  |
| 7TV Avatars | ✅ | ❌  |
| 4x Emotes | ✅ | ❌  |

*: Chatterino 2 does not yet support the new cosmetics system.

Reply:
4x Images*